code/modules/discord/discord_sql_functions.dm

```diff
@@ -0,0 +1,200 @@
+
+/**
+ * Generate a timebound token for discord verification
+ *
+ * This uses the common word list to generate a six word random token, this token can then be fed to a discord bot that has access
+ * to the same database, and it can use it to link a ckey to a discord id, with minimal user effort
```

Comment:
They're three words currently:
```suggestion
 * This uses the common word list to generate a three word random token, this token can then be fed to a discord bot that has access
 * to the same database, and it can use it to link a ckey to a discord id, with minimal user effort
```